lib/core/src/lib/login/components/login.component.ts

```diff
@@ -324,10 +324,12 @@ export class LoginComponent implements OnInit, OnDestroy {
         msg: string,
         params?: any
     ) {
-        this._message[field][ruleId] = {
-            value: msg,
-            params
-        };
+        if (field !== '__proto__' && field !== 'constructor' && field !== 'prototype') {
```

Comment:
We still want to check for `__proto__` even though it's deprecated right? Just to make myself sure :D

Reply:
Yes, even if that is deprecated, someone still can use that and assign something malicious. That solution was suggested by CodeQL too